content/en/_index.md

```diff
@@ -33,36 +29,33 @@ developers can adopt into any software update system.
 
 {{% blocks/feature icon="fa-lightbulb" title="Our work" url="docs/" %}}
 
-Discover how TUF works to secure Update systems
+Discover how TUF secures Update systems
 
 {{% /blocks/feature %}}
 
-{{% blocks/feature icon="fa-brands fa-github" title="Contribute" url="https://github.com/theupdateframework" %}}
-To contribute to The Update Framework, create a Pull request on our
-[GitHub Repositories](https://github.com/theupdateframework)
+{{% blocks/feature icon="fa-solid fa-gear" title="Production Ready" url="/docs/adoptions" %}}
+Used in production by various tech companies and open source organizations.
+
 {{% /blocks/feature %}}
 
-{{% blocks/feature icon="fa-regular fa-envelope" title="Contact Us" url="/community" %}}
-Contact us via our
-[mailing list](https://groups.google.com/forum/?fromgroups#!forum/theupdateframework).
-Questions, feedback, and suggestions are welcomed on our
-[#tuf](https://cloud-native.slack.com/archives/C8NMD3QJ3) channel on
-[CNCF Slack](https://slack.cncf.io/). {{% /blocks/feature %}}
+{{% blocks/feature icon="fa-brands fa-github" title="Contribute" url="https://github.com/theupdateframework" %}}
+Start contributing to TUF open source by creating a Pull request on
+[GitHub](https://github.com/theupdateframework) {{% /blocks/feature %}}
 
 {{% /blocks/section %}}
 
-{{% blocks/lead color="primary" %}}
+{{% blocks/lead color="primary" type="cncf" %}}
 
 **TUF** is a [Cloud Native Computing Foundation](https://www.cncf.io) (CNCF)
-Graduated project and is [used in production](docs/adoptions) by various tech
-companies and open source organizations. A variant of TUF called
-[Uptane](https://uptane.github.io/) is widely used to secure over-the-air
-updates in automobiles
+Graduated project.
 
-[![CNCF logo][]][cncf]
+[![CNCF logo][]][cncf] 
+{.cncf}
 
 [cncf]: https://cncf.io
 [cncf logo]: static/img/cncf-white.svg
 [incubating]: https://www.cncf.io/projects/
 
-{{% /blocks/lead %}}
+{{% /blocks/section %}}
+
+.
```

Comment:
I think that this change is a mistake. Shouldn't this be closing a `blocks/lead`?

```suggestion
{{% /blocks/lead %}}
```